backend/graph/api/episodes.resolvers.go

```diff
@@ -156,10 +157,46 @@ func (r *episodeResolver) Chapters(ctx context.Context, obj *model.Episode) ([]*
 	})
 	ginCtx, _ := utils.GinCtx(ctx)
 	languages := user.GetLanguagesFromCtx(ginCtx)
+
 	return lo.Map(metadataItems, func(i *common.TimedMetadata, _ int) *model.Chapter {
+		title := i.Title.Get(languages)
+		phrase, _ := r.Loaders.PhraseLoader.Get(ctx, i.ChapterType.Value)
+		switch i.ChapterType {
+		case common.ChapterTypeSong, common.ChapterTypeSingAlong:
+			if i.SongID.Valid {
+				song, _ := r.Loaders.SongLoader.Get(ctx, i.SongID.UUID)
+				if song == nil {
+					break
```

Comment:
Maybe not exclude it if the song is invalid. E.g. "andre sanger". Chapter type can still be song but it just doesn't have a number